WordPress/Classes/Services/AccountService+MergeDuplicates.swift

```diff
@@ -0,0 +1,55 @@
+import Foundation
+
+extension AccountService {
+
+    func mergeDuplicatesIfNecessary() {
+        let accounts = allAccounts()
+        guard accounts.count > 1 else {
+            return
+        }
+
+        let accountGroups = Dictionary(grouping: accounts) { $0.userID }
+        for key in accountGroups.keys {
```

Comment:
It doesn't seem like you need the key, why not `for group in accountGroups.values where group.count > 1`?

Reply:
I forget about `where` syntax sometimes 😬